CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 0.9.2 (Next)
 
+* [#167](https://github.com/slack-ruby/slack-ruby-client/pull/167): Respect rate limits during pagination by sleeping. Also add optional `pause` parameter in order to proactively sleep between each paginated request - [@jmanian](https://github.com/jmanian).
```

Comment:
Maybe simply "Added support for pausing between paginated requests that cause Slack rate limiting"?